signer/src/config.rs

```diff
@@ -0,0 +1,44 @@
+use config::{Config, ConfigError, Environment, File};
+use once_cell::sync::Lazy;
```

Comment:
I think we can use `std::sync::LazyLock` here instead. It was recently added to the standard library and I believe it has the same functionality as `once_cell::sync::Lazy`.

Reply:
Oh I did not know that. Though it looks to me like it's just available on nightly right now https://doc.rust-lang.org/std/sync/struct.LazyLock.html
Tracking issue: https://github.com/rust-lang/rust/issues/109736

Reply:
Oh you're right. I was thinking `OnceLock`, which was stabilized recently. The lazy variant was not, never mind.